po/de.po

```diff
@@ -2000,3 +2006,165 @@ msgstr "Keine Dokumente geöffnet"
+msgid "Helpful error messages in the build log."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:58
+msgid "Bug fix"
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:63
+msgid ""
+"Initial backward synctex support, improved synctex highlighting, search UI "
+"improvements, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:68
+msgid "Session management, highlight passages on build, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:73
+msgid ""
+"Zoom level popover for the .pdf preview, preview should use far less memory "
+"now, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:78
+msgid "Bug fixes"
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:83
+msgid ""
+"Enable embedded system commands, document modified on disk dialog, option to "
+"highlight current line, option to highlight matching brackets, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:88
+msgid "Add/remove packages dialog, improved HiDPI support, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:93
+msgid "Keyboard shortcuts in popovers, more commands in shortcuts bar, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:98
+msgid "More commands in shortcuts bar, autocomplete now case insensitive, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:103
+msgid "HiDPI support in .pdf preview, presistent code folding, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:108
+msgid "Initial code folding support, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:113
+msgid ""
+"References to existing labels in autocomplete, persistent master document "
+"state, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:118
+msgid "Initial spellchecking support, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:123
+msgid "Simple BibTeX wizard, more bibliography features in LaTeX editor, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:128
+msgid "BibTeX support in the build system, edit BibTeX files, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:133
+msgid "Editor settings in preferences dialog, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:138
+msgid ""
+"Build multiple times to resolve undefined references, show warnings in build "
+"log, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:143
+msgid "Indent multi-line inserts, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:148
+msgid "Multiple documents in build log, build in temporary folder, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:153
+msgid "Master document function, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:158
+msgid "New app layout, animated wizard button, big code refactor, ..."
+msgstr ""
+
+#: data/org.cvfosammmm.Setzer.appdata.xml.in:163
+msgid "Set margins in wizard, improved icon, ..."
+msgstr ""
```

Comment:
Looks like 6e7728434f1ce63ddf56981a85a3545c3a27f1b9 had no effect here :/

Reply:
Maybe try the higher level tags?

Reply:
Doens't work either, looks like a bug.